src/robot_interface/robot_interface.py

```diff
@@ -71,7 +61,7 @@ def initiate_task(self, task: Task) -> None:
             of the task until the number of allowed retries is exceeded before the task
             will be marked as failed and the mission cancelled
         NotImplementedError
```

Comment:
A few lines above this function still says "RobotInfeasibleTaskException" which I believe is no longer possible? Also you can delete this exception from robot_exception.py

Reply:
I think this is still a reasonable exception as we want to keep initiate_task for scheduling retrun to home from isar directly

Reply:
I would argue so, yes